.codespellignore

```diff
@@ -6,3 +6,4 @@ sord
 doubleClick
 sur
 jus
+euclidian
```

Comment:
It's spelled "euclidean", thus probably shouldn't be excluded

Reply:
Yep, sorry my mistake. We should correct the title and the changelog entry of the corresponding PR instead.

Reply:
Yep, sorry my mistake. We should correct the title and the changelog entry of the corresponding PR instead.